### ISSUE TITLE
PlaybyPlay Hard to Aggregate TD's accurately due to challenged plays

### ISSUE THREAD

**Issue by chrisjkeeney** [maksimhorowitz/nflscrapR#73]
To see an example do the following filters to the 2017 

filter(PlayAttempted == 1,
       touchdown == 1,
       playtype == "Pass",
       PassOutcome %in% c("Complete",NA),
       posteam == "PHI",
       Passer == "C.Wentz")

If you then sum the data, you'll get 34 TD. If you look at the stats for 2017, C.Wentz has 33 TD.  So I did some digging and found out the flagged TD was due to a pass to N.Agholor against DAL that was challenged and reversed in favor of DAL. So that touchdown shouldn't be flagged as 1. 

So I thought, no problem, just filter by reversed challenges. 

but then I added the filter(!Challenge.Replay ==1 | !chalReplayResult == "Reversed") 
and reran it and got 32 touchdowns, so one less.

Did some more digging and there was a pass to A.Jeffery against KC that was reversed in favor of PHI, so that one should be counted.

So basically we need a way of knowing if knowing a touchdown is legit due to a challenge. Seems touchdown gets flagged as 1 anytime the play is in the endzone, even for incomplete passes. 

I don't know how the scraper works, but maybe  there can be a column added called "touchdownRes" which is the result of a touchdown. 

It could be coded if the next playttempted == 1 and (!is.na(ExtraPointResult) | !is.na(TwoPointConv)) then it was a successful TD.

**Comment by nicholassharkey** [comment]
Agreed. I've been piecemeal creating a valid_play flag to try to account for these. Perhaps a more formalized heuristic for what is valid would be appreciated by the community.
